### PR TITLE
[Variadic Generics] abstract tuple of Sendable elements does not conform to Sendable

### PR DESCRIPTION
**Pull request by sophiapoirier**

### CODE REVIEW

**Comment by sophiapoirier** [comment]
@swift-ci please test

File: lib/AST/NameLookupRequests.cpp [hunk 1]
```diff
@@ -363,7 +363,7 @@ void LookupInModuleRequest::writeDependencySink(
 }
 
 //----------------------------------------------------------------------------//
-// LookupConformanceInModuleRequest computation.
+// AnyObjectLookupRequest computation.
 //----------------------------------------------------------------------------//
 
 void swift::simple_display(llvm::raw_ostream &out,
```
Comment (slavapestov) [inline lib/AST/NameLookupRequests.cpp:369]: This overload of simple_display() is for LookupConformanceInModuleRequest though. Maybe you want to add a new comment on line 379?

**Review by slavapestov — APPROVED** [review]

**Review by kavon — APPROVED** [review]

File: lib/AST/NameLookupRequests.cpp (hunk 1 above)
Comment (sophiapoirier) [reply lib/AST/NameLookupRequests.cpp:369]: sure makes sense. I just figured the comment intended to refer to what came later, but it was copy and paste duplication that was never modified, but I will move it proximate.

**Comment by sophiapoirier** [comment]
@swift-ci please test

**Pull request merged**